backtester/report/tpl.gohtml

```diff
@@ -244,265 +247,295 @@
-						</tr>
-						</thead>
-						<tbody>
-						{{ range .Statistics.FundingStatistics.Report.Items}}
-							<tr>
-								<td>{{.Exchange}}</td>
-								<td>{{.Asset}}</td>
-								<td>{{.Currency}}</td>
-								<td>{{.PairedWith}}</td>
-								<td>{{ $.Prettify.Decimal8 .InitialFunds}}</td>
-								<td>{{ $.Prettify.Decimal64 .TransferFee}}</td>
-							</tr>
-						{{end}}
-						</tbody>
-					</table>
-				</div>
-			</div>
-	{{ end }}
-		<div class="card card-cascade narrower">
-			<div class="view view-cascade bg-info">
-				<h2 id="portfolio-settings" class="px-4 card-header-title text-light">Portfolio Settings</h2>
-			</div>
-			<div class="card-body card-body-cascade ">
+			{{end}}
+			{{ if eq .Config.StrategySettings.DisableUSDTracking false}}
+				<h5>USD Totals</h5>
 				<table class="table table-hover table-bordered table-striped">
-					<thead>
+					<tbody>
 					<tr>
-						<th>Can Use Leverage</th>
-						<th>Max Leverage Rate</th>
-						<th>Max Orders With Leverage Ratio</th>
-						<th>Buy side Min Amount</th>
-						<th>Buy side Max Amount</th>
-						<th>Buy side Max Total</th>
-						<th>Sell side Min Amount</th>
-						<th>Sell side Max Amount</th>
-						<th>Sell side Max Total</th>
+						<td><b>Initial Total Funds in USD</b></td>
+						<td><b>${{  $.Prettify.Decimal2 .Statistics.FundingStatistics.Report.InitialFunds}}</b></td>
+					</tr>
+					<tr>
+						<td><b>Final Total Funds in USD</b></td>
+						<td><b>${{  $.Prettify.Decimal2 .Statistics.FundingStatistics.Report.FinalFunds}}</b></td>
 					</tr>
-					</thead>
-					<tbody>
 					<tr>
-						<td>{{.Config.PortfolioSettings.Leverage.CanUseLeverage}}</td>
-						<td>{{.Config.PortfolioSettings.Leverage.MaximumLeverageRate}}</td>
-						<td>{{.Config.PortfolioSettings.Leverage.MaximumOrdersWithLeverageRatio}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.BuySide.MinimumSize}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.BuySide.MaximumSize}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.BuySide.MaximumTotal}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.SellSide.MinimumSize}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.SellSide.MaximumSize}}</td>
-						<td>{{ $.Prettify.Decimal64 .Config.PortfolioSettings.SellSide.MaximumTotal}}</td>
+						<td><b>Difference</b></td>
+						<td><b>{{ $.Prettify.Decimal8 .Statistics.FundingStatistics.TotalUSDStatistics.HoldingValueDifference}}%</b></td>
 					</tr>
 					</tbody>
 				</table>
-			</div>
+			{{end}}
 		</div>
+	</div>
+	<div class="card card-cascade narrower">
+		<div class="view view-cascade bg-primary">
+			<h2 id="config" class="px-4 card-header-title text-light">Config</h2>
+		</div>
+	</div>
+	<div class="card card-cascade narrower">
+		<div class="view view-cascade bg-info">
+			<h2 id="strategy-settings" class="px-4 card-header-title text-light">Strategy Settings</h2>
```

Comment:
It looks like on Firefox and Chrome anywhere, the h2 padding/border extends into the tables above it which prevents people from copying its contents:

![image](https://user-images.githubusercontent.com/4685270/176370983-1ac08eca-8770-4a0b-9457-512dd14a739d.png)